src/VisualStudio/Core/Def/Implementation/Options/RoamingVisualStudioProfileOptionPersister.cs

```diff
@@ -230,7 +230,7 @@ public bool TryPersist(OptionKey optionKey, object value)
                 // We store these as strings, so serialize
                 var valueToSerialize = value as CodeStyleOption<bool>;
 
-                if (value != null)
+                if (valueToSerialize != null)
```

Comment:
instead of this:
```C#
var valueToSerialize = value as CodeStyleOption<bool>;	  
if (valueToSerialize != null)
{
    value = valueToSerialize.ToXElement().ToString();
}
```

could we do this?:
```C#
if (value is CodeStyleOption<bool> valueToSerialize )
{
    value = valueToSerialize.ToXElement().ToString();
}
```